include/wdl.h

```diff
@@ -336,6 +336,7 @@ void wdDestroyCachedImage(WD_HCACHEDIMAGE hCachedImage);
  * be used for the canvas it has been created for. */
 
 WD_HBRUSH wdCreateSolidBrush(WD_HCANVAS hCanvas, WD_COLOR color);
+WD_HBRUSH wdCreateLinearGradientBrush(WD_HCANVAS hCanvas, const WD_POINT* p0, const WD_POINT* p1, const WD_COLOR* colors, const float* offsets, UINT numStops);
```

Comment:
The need to declare the simple point structure is pain when calling the function. I would prefer if we use four `float` params instead:

```
WD_HBRUSH wdCreateLinearGradientBrush(WD_HCANVAS hCanvas, float x0, float y0, float x1, float y1, const WD_COLOR* colors, const float* offsets, UINT numStops);
```

Maybe to highlight some params are related to the stops, I would rename them to e.g. `stopColors[]`, `stopOffsets[]`. Maybe add a comment to explaing what a stop is and how to setup the `offsets[]`. It is not clear from the function prototype.

Furthermore, I am wondering whether in practice 90% of the use cases would be always using two colors and two offsets, `0.0f` and `1.0f`.

If yes, then it might be good to add a suffix "-Ex" to the function name (`wdCreateLinearGradientBrushEx()`) and adding simple wrapper function (without the current name w/o the suffix) which would have simpler interface:

```
WD_HBRUSH
wdCreateLinearGradientBrush(WD_HCANVAS hCanvas, float x0, float y0, WD_COLOR color0, float x1, float y1, WD_COLOR color1)
{
     WD_COLOR colors[] = { color0, color1 };
     float offsets[] = { 0.0f, 1.0f };
     return wdCreateLinearGradientBrushEx(hCanvasm x0, y0, x1, y1, colors, offsets, 2);
}
```

That's so simple it could be even inline in wdl.h.

Reply:
Can still move the code to the header. Since it spans multiple lines, I kept it in the source file.